src/backend/distributed/commands/table.c

```diff
@@ -1063,7 +1132,8 @@ PreprocessAlterTableStmt(Node *node, const char *alterTableCommand,
 				 */
 				constraint->skip_validation = true;
 			}
-			else if (constraint->contype == CONSTR_PRIMARY)
+			else if (constraint->contype == CONSTR_PRIMARY || constraint->contype ==
```

Comment:
I guess you forgot to use the wrapping function here :)

Reply:
Fixed.